bundles/binding/org.openhab.binding.bticino/src/main/java/org/openhab/binding/bticino/internal/BticinoDevice.java

```diff
@@ -174,12 +213,155 @@ public void receiveCommand(String itemName, Command command, BticinoBindingConfi
+                                return;
+                            } else {
+                                int RollerShutterPosStart = RollerShutterPos.get(l_where);
+
+                                if ((command instanceof PercentType)
+                                        && (ShutterBlock.getOrDefault(l_where, false) == false)) {
+                                    logger.debug("RollershutterPosStart %: {}: {}", itemName, RollerShutterPosStart);
+
+                                    int RollerShutterPosEnd = Integer.valueOf(command.toString());
+                                    if (RollerShutterPosEnd > RollerShutterPosStart) {
+                                        l_pr.addProperty("what", "2");
+                                        ShutterBlock.put(l_where, true);
+                                        m_open_web_net.onCommand(l_pr);
+                                        int RollerShutterTimeEnd = (RollerShutterPosEnd - RollerShutterPosStart)
+                                                * m_ShutterRunTime / 100;
+                                        TimeUnit.MILLISECONDS.sleep(RollerShutterTimeEnd);
+                                        l_pr.addProperty("what", "0");
+                                        m_open_web_net.onCommand(l_pr);
+                                        RollerShutterPos.put(l_where, RollerShutterPosEnd);
+                                    } else if (RollerShutterPosEnd < RollerShutterPosStart) {
+                                        l_pr.addProperty("what", "1");
+                                        ShutterBlock.put(l_where, true);
+                                        m_open_web_net.onCommand(l_pr);
+                                        int RollerShutterTimeEnd = (RollerShutterPosStart - RollerShutterPosEnd)
+                                                * m_ShutterRunTime / 100;
+                                        TimeUnit.MILLISECONDS.sleep(RollerShutterTimeEnd);
+                                        l_pr.addProperty("what", "0");
+                                        m_open_web_net.onCommand(l_pr);
+                                    }
+                                    RollerShutterPos.put(l_where, RollerShutterPosEnd);
+                                    eventPublisher.postUpdate(itemName,
+                                            PercentType.valueOf(Integer.toString(RollerShutterPosEnd)));
+                                    logger.debug("RollershutterPosEnd %: {}: {}=>{} / {}", itemName,
+                                            RollerShutterPosStart, RollerShutterPosEnd,
+                                            Integer.toString(RollerShutterPosEnd));
+                                    ShutterBlock.put(l_where, false);
+                                    return;
+                                } else if (ShutterBlock.getOrDefault(l_where, false) == false) {
```

Comment:
Make this an `if` on a new line/paragraph. It's not an `else if`

Reply:
OK
see commit https://github.com/openhab/openhab1-addons/pull/5486/commits/331f01fbd97de4a8941d866206d522d478e12bb8

Reply:
I reverted the change https://github.com/openhab/openhab1-addons/commit/331f01fbd97de4a8941d866206d522d478e12bb8
as for some reason it gives an error